src/Form.js

```diff
@@ -15,25 +17,38 @@ const configureDate = (date) => {
 const submissionType = (e) => {
   console.log(location.pathname)
   if(location.pathname === '/') {
+    setLoading(true)
    return getCoords(e, userDate)
   } else {
    return fetchSpecific(e)
   }
 }
 
+const maxDateMaker = () => {
+  if(location.pathname === '/') {
+    setFormDate(dayjs().subtract(7, 'month').format('YYYY/MM/DD').split('/').join('-'))
+  } else {
+    setFormDate(dayjs().format('YYYY/MM/DD').split('/').join('-'))
+  }
+}
+
+useEffect(() => {
+  console.log('hello form')
+  console.log(formDate)
+  maxDateMaker()
+},[maxDateMaker, formDate])
```

Comment:
Nice job getting everyting done in this one maxdateMaker function. You may want to look into if you actually need a useEffect hook here- you might be able to do this functionality in just another function.